fe1-web/components/TextInputList.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+import RemovableTextInput from './RemovableTextInput';
+
+/**
+ * Component which creates a list of Text input fields which can be deleted individually
+ * Returns array with non-empty Values of the text fields
+ * It uses the RemovableTextInput.tsx component to show the individual inputs
+ * Input: A function that stores the values that were inputted by the user
+ * Output: Array of the unique, nonempty values inputted by the user
+ */
+
+const TextInputList = (props: IPropTypes) => {
+  const { onChange } = props;
+  const [idCount, setIdCount] = useState(1);
+  const [userOptions, setUserOptions] = useState([{ id: 0, value: '' }]);
+
+  const updateParent = (options: { id: number, value: string }[]) => {
+    // Gets the distinct options which are not empty ('')
+    // Set() keeps the order the same
```

Comment:
thanks, I learned something 😄

Reply:
Yes, in python and C I think it messes the order up but in JS it doesn't :)